response_operations_ui/error_handlers.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from requests import RequestException
+from structlog import wrap_logger
+
+from response_operations_ui import app
+from response_operations_ui.exceptions.exceptions import ApiError
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+@app.errorhandler(ApiError)
+def api_error(error):
+    logger.error('Api failed to retrieve required data',
```

Comment:
It might be worth logging with logger.exception to capture the traceback

Reply:
Is there any useful traceback coming from here? We log out the url we tried to hit and the status code that we got back, is there any benefit in seeing the code which triggered this?

Reply:
I guess it's useful debug context to figure out the path the code followed before getting an error. It's probably fine for now because the api will only be called from one place but if we got to a point where the api was being called in different places it would definitely aid the debugging process

Reply:
I've added it in. Instead of logging at exception level I've used the `format_exc_info` processor to include it in a error level log

Reply:
👍 but I thought you still had to `log.exception` to see it. @JamesGardiner do you know from when you did that little test?

Reply:
You can see it from this, If you run the tests you can see it in the logs